.travis.yml

```diff
@@ -75,22 +75,19 @@ script:
           /bin/bash -c "GOARCH=amd64 PASSES='fmt bom dep' ./test.sh"
         ;;
       linux-amd64-integration-1-cpu)
-        # TODO: Reenable 'race' when https://github.com/etcd-io/etcd/issues/12336 fixed.
         docker run --rm \
           --volume=`pwd`:/go/src/go.etcd.io/etcd gcr.io/etcd-development/etcd-test:go${TRAVIS_GO_VERSION} \
-          /bin/bash -c "GOARCH=amd64 CPU=1 PASSES='integration' RACE='false' ./test.sh"
+          /bin/bash -c "GOARCH=amd64 CPU=1 PASSES='integration' RACE='true' ./test.sh"
```

Comment:
Let's test only with race the 4 CPUs setup and grpcproxy tests. I would keep 1,2 CPU on false.

Motivation: This makes the test significantly slower and we have small 'freebee' quota on Travis for 5 workers only.

Reply:
Makes sense! Made the adjustments, please have a look. 

I also let it run before and it was green the first time around FYI -> https://travis-ci.com/github/etcd-io/etcd/builds/223181398